tests/post_training/pipelines/base.py

```diff
@@ -367,6 +368,11 @@ def save_compressed_model(self) -> None:
             )
             self.path_compressed_ir = self.output_model_dir / "model.xml"
             ov.serialize(ov_model, self.path_compressed_ir)
+        elif self.backend == BackendType.FX_TORCH:
+            exported_model = torch.export.export(self.compressed_model, (self.dummy_tensor,))
```

Comment:
Compressed FX model can not be converted to OV model directly?

Reply:
Yes, the error is
```
 Failed: Couldn't get TorchScript module by tracing. With exception:
E           
E                   Calling train() or eval() is not supported for exported models.
E                   Please call `torch.ao.quantization.move_exported_model_to_train(model)` (or eval) instead.
E           
E                   If you cannot replace the calls to `model.train()` and `model.eval()`, you may override
E                   the behavior for these methods by calling `torch.ao.quantization.allow_exported_model_train_eval(model)`,
E                   which does the above automatically for you. Note that this has limited effect on switching
E                   behavior between train and eval modes, and should be used only for special ops such as dropout
E                   and batchnorm.

```